video/zimg.c

```diff
@@ -548,8 +548,8 @@ static bool mp_zimg_state_init(struct mp_zimg_context *ctx,
         params.allow_approximate_gamma = 1;
 
     // leave at default for SDR, which means 100 cd/m^2 for zimg
-    if (ctx->dst.color.sig_peak > 0 && mp_trc_is_hdr(ctx->dst.color.gamma))
-        params.nominal_peak_luminance = ctx->dst.color.sig_peak * MP_REF_WHITE;
+    if (ctx->dst.color.hdr.max_luma > MP_REF_WHITE && mp_trc_is_hdr(ctx->dst.color.gamma))
```

Comment:
pretty sure this should still be `> 0`

Reply:
Ok. I actually was thinking what was the intended outcome, but let's keep it as 0.